man/systemd.exec.xml

```diff
@@ -248,6 +248,42 @@
         <xi:include href="system-only.xml" xpointer="singular"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>MountPaths=</varname></term>
+
+        <listitem><para>This setting is similar to <varname>RootImage=</varname> in that it mounts a file
+        system hierarchy from a block device node or loopback file, but the destination directory can be
+        specified as well as mount options. This option expects a whitespace separated list of mount
```

Comment:
hmm, if this is about block-based file systems only (i.e. not NFS or procfs/sysfs/ style things), then maybe call this MountImage=?

I mean, I think restricting this to disk images makes a ton of sense.

Reply:
Sorry, not following - you mean rename from MountImages= to MountImage= ? But it accepts multiple values, so I thought the plural made sense?

Reply:
Or do you mean in the comment? Eg:

> tuple of source path

 to

> tuple of source path (pointing to disk images)

Reply:
hmm, ignore me, got confused because MountPaths= is mentioned in the top of the issue

Reply:
it's my bad, if one reviews commit by commit it was pretty confusing, fixed now with a squash and giving credit via git commit message instead